lua/autorun/client/cl_nadmodpp.lua

```diff
@@ -32,7 +68,14 @@ hook.Add("HUDPaint", "NADMOD.HUDPaint", function()
 	if !tr.HitNonWorld then return end
 	local ent = tr.Entity
 	if ent:IsValid() && !ent:IsPlayer() then
-		local text = "Owner: " .. (Props[ent:EntIndex()] or "N/A")
+		local index = ent:EntIndex()
```

Comment:
You'll also want to show 'Owner: Ownerless' + World. 
I also fear this approach means you can't see owner nicknames upon them disconnecting. One solution might be keeping a serverside + clientside mapping of steamID: Last Known Nickname, and falling back to that if they're not ingame anymore.

Also `index` is unused.